src/simulation/atmos_grids.jl

```diff
@@ -0,0 +1,701 @@
+# This file contains the definitions of common AbstractAtmosGrids.
```

Comment:
In terms of scope, these functions are just wrappers around ClimaCore objects, which leads me to think: should we just define these helper functions in ClimaCore? (otherwise we may end up repeating these patterns in the land) cc @simonbyrne